internal/service/networkcontainer/resource_network_container.go

```diff
@@ -51,6 +50,7 @@ func Resource() *schema.Resource {
 				Type:         schema.TypeString,
 				Optional:     true,
 				Default:      constant.AWS,
+				ForceNew:     true,
```

Comment:
provider change forces a replace

Reply:
Just to understand the why behind this change, the API does not support updating the providerName?

Reply:
i was under that impression but i checked again and it looks like it's allowed (although it's a bit tricky because they need different attributed).
updated